data/json/monsters/fungus_zombie.json

```diff
@@ -42,6 +42,36 @@
     "relative": { "melee_dice_sides": 3, "armor": { "bash": 1, "electric": 2 } },
     "extend": { "species": [ "FUNGUS" ], "special_attacks": [ [ "FUNGUS", 200 ] ] }
   },
+  {
+    "id": "mon_gasbag_dusted",
+    "type": "MONSTER",
+    "name": { "str": "dusted gasbag zombie" },
+    "description": "This grotesquely swollen corpse has been heavily dusted with a thick gray powder.",
+    "copy-from": "mon_zombie_gasbag",
+    "default_faction": "zombie_dusted",
+    "color": "black_yellow",
+    "vision_day": 10,
+    "upgrades": { "age_grow": 1, "into": "mon_gasbag_fungalize" },
+    "proportional": { "speed": 0.6 },
+    "relative": { "melee_skill": -1, "melee_dice": -1 },
+    "extend": { "flags": [ "NO_FUNG_DMG" ] }
+  },
+  {
+    "id": "mon_gasbag_fungalize",
+    "type": "MONSTER",
+    "name": { "str": "bloated fungal stalk zombie" },
```

Comment:
While I have nothing against the name itself, it's very long. Our compass and sidebar sizes ain't adapted to such long names. I think 3-words-name is maximum we can allow? Maybe invent some short yet memorable name? I mean, we don't call a `boomer` "big bloated explosive zombie", right?
Here and below.